### ISSUE TITLE
**Report TODO list**

### ISSUE THREAD

**Issue by miccio-dk** [Christie22/SingleChannelDenoising#3]
1. [ ] Introduction `C`
2. [ ] Related work `R`
    1. [ ] Classical Approaches 
    2. [ ] Deep Learning `R`
    3. [ ] State of the art 
3. [x] Methodology `R`
    1. [ ] Deep Learning `C`
        1. [ ] Multilayer Perceptron `C`
        2. [ ] Autoencoders (AE) `C`
        3. [ ] Convolutional Neural Networks (CNN) `C`
        4. [ ] Recurrent Neural Networks (RNN) `R`
        5. [ ] Activation `R`
        6. [ ] Batch Normalization `C`
        7. [ ] Dropout `C`
        8. [ ] Optimization `C`
    2. [ ] Data Representation `C`
        1. [ ] Time Domain Representation `C`
        2. [ ] Discrete Fourier Transform (DFT) `C`
        3. [ ] Short-Time Fourier Transform (STFT) `C`
    3. [ ] Processing `R`
        1. [ ] Dynamic Range Compression `R`
        2. [ ] Mel-Frequencies `R`
    4. [ ] Evaluation `C`
        1. [ ] Signal-to-Noise Ratio (SNR) `C`
        2. [ ] Source Evaluation `?`
        3. [ ] STOI `C`
        4. [ ] PESQ `C`
4. [x] Experimental Setup `R`
    1. [x] Dataset `R`
    2. [x] Framework `R`
        1. [ ] Processing `R`
        2. [ ] Training `R`
        3. [ ] Results `R`
        4. [ ] Denoising `R`
    3. [ ] Models `R`
        1. [ ] Convolutional Autoencoder (Baseline) `R`
        2. [ ] Convolutional-Recurrent Model `R`
        3. [ ] Temporal Convolutional Network `R`
    4. [ ] Experiments `?`

**Comment by miccio-dk** [comment]
### Figure conventions: 
- In python: 
```py
plt.figure(figsize=(16, 4), dpi=300)
```
- In LaTeX:
```latex
\begin{figure}[h]
    \centering
    \includegraphics[width=0.75\textwidth]{figures/fig_name}
    \caption{caption text, starting in lowercase}
    \label{fig_name}
\end{figure}
```
